### ISSUE TITLE
Export to Excel

### ISSUE THREAD

**Issue by christoffer** [christoffer/readtracker#8]
Add export/import to Excel spread sheet support (for offline usage)

**Comment by anthonymonori** [comment]
Would that be convenient at all for the users? Do you mean comma-separated values file?

**Comment by takieddine01** [comment]
This is a good and an useful feature

**Comment by christoffer** [comment]
Nah, this was a feature request by a user on Google Play. So I added it here so I wouldn't forget it.

CSV, while providing the same functionality, is a little less accessible for most users since it requires additional steps to open up in excel. So exporting to (some very early/primitive) version of Excel could make sense, but it's not a high prio at all for me personally.
